content/en/real_user_monitoring/guide/session-replay-getting-started.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Getting started with Session Replay
+kind: guide
+description: Guide for enabling Session Replay and how to set privacy options
+beta: true
+further_reading:
+- link: '/real_user_monitoring/explorer'
+  tag: 'Documentation'
+  text: 'Visualize your RUM data in the Explorer'
+---
+
+## What is Session Replay?
+
+Session Replay expands your UX monitoring by allowing you to capture and visually replay the web browsing experience of your users.
+Used in combination with RUM performance data, Session Replay can be an asset for error identification, reproduction & resolution, and can bring invaluable insights into your web application’s usage patterns and design pitfalls.
+
+## Collect Session Replay data
+
+### Initial setup
+
+To use Session Replay, Datadog RUM Browser Monitoring must already be set up, following the steps detailed [here][1]: application creation, client token generation, and RUM SDK setup.
+
+Session Replay is available through a dedicated build of the RUM Browser SDK. As such, enabling Session Replay only requires changing the npm package name or CDN URL, depending on your chosen installation method.
+
+#### npm
+The `@datadog/browser-rum package` is to be replaced with `[@datadog/browser-rum-recorder][2]` When `datadogRum.init()` is called, the Session Replay recording is started as well.
+
+``` javascript
+import { datadogRum } from '@datadog/browser-rum-recorder'
+
+datadogRum.init({
+  applicationId: '<DATADOG_APPLICATION_ID>',
+  clientToken: '<DATADOG_CLIENT_TOKEN>',
+  site: '<DATADOG_SITE>',
+  //  service: 'my-web-application',
+  //  env: 'production',
+  //  version: '1.0.0',
+  sampleRate: 100,
+  trackInteractions: true,
+})
+```
+
+#### CDN
+The Browser SDK URL https://www.datadoghq-browser-agent.com/datadog-rum.js is to be replaced with https://www.datadoghq-browser-agent.com/datadog-rum-recorder.js When `DD_RUM.init()` is called, the Session Replay recording is started as well.
+
+*Supported browsers*: The Session Replay recorder supports all the browsers supported by the RUM Browser SDK with the exception of IE11. See the [browser support table][3].
+
+#### Configuration
+
+The usual [RUM initialization parameters][4] are all supported. 
+
+In addition, the Session Replay recorder accepts a `manualSessionReplayRecordingStart` option. If set to `true`, the Session Replay recording will not start automatically when calling `init()`. To then start the recording, call `startSessionReplayRecording()`. This can be useful to conditionally start the recording (e.g, to only record authenticated user sessions):
```

Comment:
```suggestion
In addition, the Session Replay recorder accepts a `manualSessionReplayRecordingStart` option. If set to `true`, the Session Replay recording does not start automatically when calling `init()`. To then start the recording, call `startSessionReplayRecording()`. This can be useful to conditionally start the recording (for example to only record authenticated user sessions):
```